tests/fuzz/analyzer_fuzzer.go

```diff
@@ -93,7 +93,7 @@ func runAnalyzer(sa *local.IstiodAnalyzer) (local.AnalysisResult, error) {
 }
 
 // FuzzAnalyzer implements the fuzzer
-func FuzzAnalyzer(data []byte) int {
+func FFuzzAnalyzer(data []byte) int {
```

Comment:
Why is this changed?

Reply:
Marking as solved because of [this comment](https://github.com/istio/istio/pull/35900#issuecomment-971786345).